### ISSUE TITLE
tss makestyles getting overridden by mui styles

### ISSUE THREAD

**Issue by asimyaramis** [garronej/tss-react#159]
Hi There,

I am trying migrate a big project mui v4 to v5 we had lots of makestyles in our codebase that is why i choose tss-react. But after that I realized that my styles getting overriden by mui and for my case it is not possible to increase the css specifity where the visual bugs happen bc project is wide. I looked it up there are several issues on this repo and mui repo but there is no specific solution.

![image](https://user-images.githubusercontent.com/118752947/224896885-55b9cd4b-f5aa-486b-ba3a-9036c6c91c1a.png)

![image](https://user-images.githubusercontent.com/118752947/224896973-5f6202a1-0703-42c6-8d01-e5f23b3e6312.png)

Dependencies :
"@mui/codemod": "^5.11.12",
"@mui/icons-material": "^5.11.11",
"@mui/lab": "^5.0.0-alpha.122",
"@mui/material": "^5.11.12",
"@mui/styles": "^5.11.12",
"@mui/x-data-grid-premium": "^5.17.8",
"@mui/x-date-pickers": "^5.0.0",
"@mui/x-date-pickers-pro": "^5.0.0",
"tss-react": "^4.7.2",

and I don't want this media queries also
![image](https://user-images.githubusercontent.com/118752947/224897810-c8644c4b-eb3d-4351-9bb4-2231be16ddd9.png)

Thanks in advance

**Comment by garronej** [comment]
Hi,  
You'll have to be way more specific.  
Maybe produce a reproduction repo where you show what you expect vs what you get.  

My guess is you're still using clsx instead of cx.

**Comment by asimyaramis** [comment]
Unfortunately I can not provide a demo repo. But I added two screenshot before one is the expected behaviour second one is the tss one. I removed all material libraries also. Somehow media query take precedence on tss it seems like. I am not using clsx. When migrating I used codemod jss-to-tss-react.
<<before>>
![image](https://user-images.githubusercontent.com/118752947/225014046-4d3344c2-de55-4431-9cd2-1e8e3ef23458.png)

<<after>>
![image](https://user-images.githubusercontent.com/118752947/225014310-90506d9b-00f3-4dfa-984a-f0272527636c.png)
